array_api_compat/torch/__init__.py

```diff
@@ -0,0 +1,19 @@
+from torch import *
+
+# Several names are not included in the above import *
```

Comment:
Ho that is bad. I guess this is because we're missing things in the `__all__` attribute? If you have a list for these, could you open an issue on the pytorch repo for us to fix them?

Reply:
Ho I forgot about that issue. Perfect! Thanks!